### PR TITLE
daemon.start: add lxcfs.debug config option

### PR DESCRIPTION
**Pull request by mihalicyn**
Signed-off-by: Alexander Mikhalitsyn <aleksandr.mikhalitsyn@canonical.com>

### CODE REVIEW

**Comment by stgraber** [comment]
Needs to be added to snapcraft.yaml too

**Comment by mihalicyn** [comment]
> Needs to be added to snapcraft.yaml too

ugh, right. Thanks!

File: snapcraft.yaml [hunk 1]
```diff
@@ -43,6 +43,7 @@ description: |-
    - lxcfs.pidfd: Start per-container process tracking [default=false]
    - lxcfs.loadavg: Start tracking per-container load average [default=false]
    - lxcfs.cfs: Consider CPU shares for CPU usage [default=false]
+   - lxcfs.debug: Enable lxcfs debug mode (passes -d option) [default=false]
```
Comment (stgraber) [inline snapcraft.yaml]: I'd probably match the LXD one and say `Increase logging to debug level`.
The user doesn't know or care that it's the `-d` flag to LXCFS and lxcfs is already implied by the option being called `lxcfs.debug` :)

Comment (mihalicyn) [reply snapcraft.yaml]: fixed! :-)

**Pull request merged**